main.go

```diff
@@ -301,20 +297,15 @@ func extractMachineReadable(args []string) ([]string, bool) {
 
 func loadConfig() (*config, error) {
 	var config config
-	config.Plugins = plugin.Config{
-		PluginMinPort: 10000,
-		PluginMaxPort: 25000,
+	config.Plugins = &packer.PluginConfig{
+		PluginMinPort:      10000,
+		PluginMaxPort:      25000,
+		KnownPluginFolders: packer.PluginFolders("."),
```

Comment:
this is cool. We should probably also use knownpluginfolders for json configs too

Reply:
Yes !